src/test/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApprovalTest.java

```diff
@@ -179,4 +178,19 @@ public boolean handleConfirm(Page page, String message) {
         assertNotNull(page.getElementById(String.format("pcp-%s", CLASSPATH_HASH5)));
     }
 
+    private void clickAndWait(HtmlElement e) throws IOException {
+        e.click();
+        e.getPage().getWebClient().waitForBackgroundJavaScript(10000);
+    }
+
+    private void click(HtmlPage page, String id, String value) throws IOException {
+        for (HtmlElement e : page.getElementById(id).getElementsByTagName("button")) {
+            if (e.hasAttribute("class") && value.equals(e.getAttribute("class"))) {
```

Comment:
iirc it also tests it is removed from the pending list...

In any case the changes only tried to make it compatible with the upgraded HTMLUnit, not to refactor the test.

Thanks!